src/components/profile/ProfileOtherAdminCard.tsx

```diff
@@ -0,0 +1,246 @@
+              memberPhone: Object.keys(formValues)
+                .filter(propertyId => propertyId === 'phone')
+                .map(() => ({
+                  member_id: memberId,
+                  phone: formValues['phone'],
+                })),
+            },
+          })
+            .then(() => {
+              message.success(formatMessage(commonMessages.event.successfullySaved))
+              refetchProperties()
+            })
+            .catch(handleError)
+
+          delete formValues['phone']
+        }
+        updateMemberProperty({
+          variables: {
+            memberId,
+            memberProperties: Object.keys(formValues)
+              .filter(propertyId => formValues[propertyId])
+              .map(propertyId => ({
+                member_id: memberId,
+                property_id: propertyId,
+                value: formValues[propertyId],
+              })),
+          },
+        })
+          .then(() => {
+            message.success(formatMessage(commonMessages.event.successfullySaved))
+            refetchProperties()
+          })
+          .catch(handleError)
+          .finally(() => setLoading(false))
+      }
+    })
+  }
+  return (
+    <AdminCard {...cardProps}>
+      <Typography.Title className="mb-4" level={4}>
+        {formatMessage(messages.otherInfoTitle)}
+      </Typography.Title>
+      <StyledForm
+        labelCol={{ span: 24, md: { span: 4 } }}
+        wrapperCol={{ span: 24, md: { span: 8 } }}
+        onSubmit={handleSubmit}
+      >
+        {phoneEnableStatus ? (
+          <Form.Item label={formatMessage(messages.phone)}>
+            {form.getFieldDecorator(`phone`, {
+              initialValue: phoneNumber,
+              validateTrigger: 'onSubmit',
+              rules: [
+                {
+                  required: true,
+                  message: formatMessage(messages.enterPhone),
+                },
+                {
+                  validator: (rule, value, callback) => {
+                    const regex = /^\+?\(?[0-9]+\)?-?[0-9]+$/g
+                    if (!form.getFieldValue('phone').match(regex)) {
+                      callback(formatMessage(messages.enterPhone))
+                    } else {
+                      callback()
+                    }
+                  },
+                },
+              ],
+            })(<MigrationInput type="phone" />)}
+          </Form.Item>
+        ) : (
+          <></>
+        )}
+        {properties.map(item => {
+          let defaultValue = item.memberProperties.find(item => item.member_id === memberId)?.value || ''
+          console.log('defaultValue', defaultValue)
+          return (
+            <Form.Item label={`${item.name}`}>
+              {form.getFieldDecorator(`${item.id}`, {
+                initialValue: defaultValue,
+                rules: [
+                  {
+                    required: true,
+                    message: `${formatMessage(messages.enter)}${item.name}`,
```

Comment:
it will display below image
could refer `ResetPasswordPage:166` and `pageMessage/translations:124` file
<img width="628" alt="image" src="https://user-images.githubusercontent.com/28040359/215982103-a8caff7e-d433-4982-ab65-5f83d2cfd2f7.png">